earthfile2llb/interpreter.go

```diff
@@ -76,12 +76,31 @@ func (i *Interpreter) Run(ctx context.Context, ef spec.Earthfile) (err error) {
 	return i.errorf(ef.SourceLocation, "target %s not found", i.target.Target)
 }
 
+func (i *Interpreter) isPipelineTarget(ctx context.Context, t spec.Target) bool {
+	if len(t.Recipe) == 0 {
+		return false
+	}
+
+	for _, stmt := range t.Recipe {
+		if stmt.Command != nil && stmt.Command.Name == "PIPELINE" {
+			return true
+		}
+	}
```

Comment:
Should we just look at the first command only? I think the intention is for `PIPELINE` to forever remain the first command in the recipe.

Reply:
I thought this was a good way to trigger the pipeline code path and later validate the order. That way, the validation code lives in one place.

Reply:
True - that works I guess. The if statement before the `for` seems unnecessary then?